doc/managing-certificates-for-application-ingress.md

```diff
@@ -0,0 +1,34 @@
+# Managing certificates for application ingress
+
+For each domain in CF:
+
+0. [Make sure SDS is enabled for Istio](https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-tls-ingress-gateway-using-sds)
```

Comment:
Is this enabled by default in cf-for-k8s?  If not, could we do that as part of this story, in order to remove this complexity from the operator?

Reply:
@XanderStrike has some concerns about SDS due to its alpha status:
https://istio.io/about/feature-stages/#security-and-policy-enforcement

We think we might need to evaluate what the implications of that are before recommending this as a default configuration.

Reply:
Copying in a thing I said in slack:
For what it's worth, I see
> Cert management on Ingress Gateway

is listed as

> Beta

And that link takes me to the SDS section

Reply:
Whatever we decide, lets try to ensure it only relies on features we've got enabled by default in cf-for-k8s

Reply:
We've submitted a PR!
https://github.com/cloudfoundry/cf-k8s-networking/commit/592babdf2842d4e46797fd6782b1215bd8b7fb6b

🥳

Reply:
@tcdowney PR is merged.

Reply:
Yep, let's delete this step!

Reply:
Deleted :)
https://github.com/cloudfoundry/cf-k8s-networking/pull/28/commits/607868468f156546c37bf43761a66b0a2d745694#diff-612cba4c2e05c8faf18af9cba41df2c3L5